### PR TITLE
python 3 compat

### LINKED ISSUE #29: _convert_rgb uses integer division
[This](https://github.com/mapbox/rio-alpha/blob/master/rio_alpha/utils.py#L64):

```
def _convert_rgba(rgb_orig):
    ...
    mod = int(math.ceil(rgb_orig[:, :, 0].shape[min_dimension] / 200))
```

won't work properly on Python 2 because of integer division. 

```
$ python
Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 12:54:16)
>>> import math
>>> math.ceil(700 / 200)
3.0
```

**The fix** is to use the [new division operator](https://www.python.org/dev/peps/pep-0238/). It's a good idea to put `from __future__ import division` at the top of every Python file that does division with anything that could potentially be an `int`:

```
>>> from __future__ import division
>>> math.ceil(700 / 200)
4.0
```

Python 3 does the right thing by default
```
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  5 2015, 21:12:44)
>>> import math
>>> math.ceil(700 / 200)
4
```

This breaks a couple tests so they'll need to be updated.

### PR DESCRIPTION
**Pull request by perrygeo**
* Implicit relative imports are ambiguous and aren't allowed in Python 3.
* zip objects must be explicitly iterated over to realize their values
* removing extra imports for names that aren't used in the module
* integer division is problematic, use new division operator (fixes #29 )

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 machine:
     python:
-        version: 2.7.11
-  # post:
-  #   - pyenv global 2.7.11 3.4.3
+        # version: 2.7.11
+        version: 3.5.1
```
Comment (virginiayung) [inline circle.yml]: Trying to get circleci to run multiple versions of python was a huge pain. I tried all following 2 suggestions and neither worked:
    1. circleci docs: https://circleci.com/docs/language-python/
    2. tox plugin: https://github.com/samstav/circleci-python-sandbox
 
Also, circleci uses pyenv to install all [other python versions](https://circleci.com/docs/build-image-trusty/#python) other than the default: `2.7.11`. When running matplotlib in pyenv virtualenv, the GUI frameworks that matplotlib uses for interactive figures have some issues with the virtual environments. http://matplotlib.org/faq/virtualenv_faq.html. 
screen shot 2016-09-19 at 6 08 47 pm
Switching to travis for now!

**Comment by perrygeo** [comment]
😬  @virginiayung - sorry about the CircleCI pains. Speaking as someone with little CircleCI experience, Travis seems considerably easier to work with regarding complex build matrices. So switching makes sense for now.

Eventually I'd like to get CircleCI able to support all different versions of Rasterio/GDAL/Python because that is really critical to ensuring future and backwards compatibility of our plugins. Testing all permutations locally is not sustainable!

File: circle.yml (hunk 1 above)
Comment (perrygeo) [inline circle.yml]: 👍  Anything else before we 🚢 ?

Comment (virginiayung) [inline circle.yml]: We're good to go!

Comment (virginiayung) [inline circle.yml]: Thanks for all your work here @perrygeo!

**Pull request merged**